beacon-chain/sync/subscriber.go

```diff
@@ -334,7 +332,6 @@ func (s *Service) wrapAndReportValidation(topic string, v wrappedVal) (string, p
 					"topic":        topic,
 					"multiaddress": multiAddr(pid, s.cfg.p2p.Peers()),
 					"peerID":       pid.String(),
-					"agent":        agentString(pid, s.cfg.p2p.Host()),
```

Comment:
why is this removed ? its very useful on mainnet to catch misbehaving clients

Reply:
The reason of removal is written in the corresponding [commit message](https://github.com/prysmaticlabs/prysm/pull/14475/commits/55db3511022b14e43ad65d8c61240db4ae56f034).

I removed it because it was quite useless on devnet, since we know who is who.
But yes, on mainnet/testnet, it makes sense.

Fixed in 0cf7d4fcdd5560611ba48ece95903fdeefaded02.